src/server/graphics/multiplexing_display.cpp

```diff
@@ -319,5 +322,21 @@ void mg::MultiplexingDisplay::resume()
 
 auto mg::MultiplexingDisplay::create_hardware_cursor() -> std::shared_ptr<Cursor>
 {
-    return {};
+    std::vector<Display*> platform_displays;
+    for (auto& display : displays)
+    {
+        platform_displays.push_back(display.get());
+    }
+    try
+    {
+        return std::make_shared<MultiplexingCursor>(platform_displays);
+    }
+    catch (std::exception const&)
+    {
+        mir::log(
+            mir::logging::Severity::informational,
+            "display",
+            "Failed to create hardware cursor");
+        return nullptr;
```

Comment:
Not sure why, but I prefer default constructing a `shared_ptr<...>` (original code) to implicitly constructing it with `nullptr`.